docs/spec/resolution.md

```diff
@@ -33,7 +33,7 @@
             1. Set the _Next Update Commitment_ value to `null`.
             2. Skip all further Create operation processing steps and proceed to [post-Create operation processing](#post-create-operation-processing).
     3. Validate the [_Chunk File Delta Entry_](#chunk-file-delta-entry). If the [_Chunk File Delta Entry_](#chunk-file-delta-entry) is valid, proceed, if the Entry is invalid, let the state of the DID reflect an _Empty DID State_.
-    4. Generate a hash of the decoded [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [Hashing Process](#hashing-process) and ensure the hash equals the value of the [_Create Operation Suffix Data Object_](#create-suffix-data-object) `delta_hash` property. If the values are exactly equal, proceed, if they are not:
+    4. Generate a hash of the canonicalized [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [`HASH_PROTOCOL`](#hash-protocol) and ensure the hash equals the value of the [_Create Operation Suffix Data Object_](#create-suffix-data-object) `delta_hash` property. If the values are ****exactly**** equal, proceed, if they are not:
```

Comment:
this is not accurate description of the reference implementation behavior... zlib compression is applied here, not its not mentioned in the spec... presence or absence of compression will yield different CAS identifiers.

Reply:
TODO: No mention of compression, need to be specific about what compression algo is being used

Reply:
TODO: pin reference to a specific implementation of compression/algo

Reply:
Still debating which one to use, we should move this to a separate discussion.